src/main/java/org/sagebionetworks/bridge/exporter/helper/BridgeHelper.java

```diff
@@ -49,6 +53,23 @@ public void completeUpload(String uploadId) {
     }
 
     /**
+     * Helper method to update export status in records
+     */
+    public void updateRecordExporterStatus(List<String> recordIds, RecordExportStatusRequest.ExporterStatus status) {
+        // first create a credentials for this session
+        SpringConfig config = new SpringConfig();
+        SignInCredentials credentials = config.bridgeWorkerCredentials();
+        setCredentials(credentials);
+
+        // then update status
+        RecordExportStatusRequest request = new RecordExportStatusRequest(recordIds, status);
+        sessionHelper(() -> {
+            session.getWorkerClient().updateRecordExporterStatus(request);
+            return null;
+        });
```

Comment:
List of record IDs may be larger than your batch size. You'll need to break down the list into batches and call Bridge in a loop, sleeping between each batch. (Most likely sleeping for 1 second between each batch.)